src/components/DayPickerRangeController.jsx

```diff
@@ -41,6 +41,7 @@ const propTypes = {
   withPortal: PropTypes.bool,
   hidden: PropTypes.bool,
   initialVisibleMonth: PropTypes.func,
+  scrollable: PropTypes.bool,
```

Comment:
This might make the orientation idea a bit sketchy since the DRP and SDP take an orientation prop